src/web-vitals-autoinstrumentation.ts

```diff
@@ -20,12 +20,10 @@ import {
   TTFBAttribution,
   TTFBMetricWithAttribution,
 } from 'web-vitals/attribution';
-import {
-  InstrumentationBase,
-  InstrumentationConfig,
-} from '@opentelemetry/instrumentation';
+import { InstrumentationConfig } from '@opentelemetry/instrumentation';
 import { Span } from '@opentelemetry/api';
 import { VERSION } from './version';
+import { InstrumentationAbstract } from '@opentelemetry/instrumentation/build/src/instrumentation';
```

Comment:
because it's just a type, does it make any sense to copy-paste the lil dude & define it inline of this file instead of importing it from the nested directory structure? might be a cursed and toxic idea

Reply:
@ahrbnsn addressed here 84008e9f0b7551b5938e84feeb7a06c5216fe1bd